tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -1887,6 +1912,11 @@ private void VerifyDelete(int tournamentId, Times times)
             _tournamentServiceMock.Verify(ts => ts.Delete(tournamentId), times);
         }
 
+        private void VerifyActivateTournament(int tournamentId, Times times)
```

Comment:
What is the difference between this and line 1789?

Reply:
1789 sets up mock method.
![image](https://user-images.githubusercontent.com/11539042/37581120-bdd05260-2b4f-11e8-97fb-731bc83ee5a1.png)

1915 verifies if the method in mock was call specific amount of times.
![image](https://user-images.githubusercontent.com/11539042/37581160-fbc08176-2b4f-11e8-9f63-d4e7ca4594b5.png)

Reply:
Sorry - missed that

Reply:
It's ok.